neural-network/PulsedRADAR/matchedFilter.py

```diff
@@ -0,0 +1,133 @@
+# Example to apply matched filter to simulated RADAR data generated by Moku:Go and Received / de-noised by Moku:Pro
+# This will apply a matched filter of a known pulse width and carrier frequency to simulated received
+# radar data from the Moku:Pro.  In this example we add noise and then use an autoencoder with the 
+# Moku Neural Network instrument to de-noise the data.  We then apply a matched filter to the
+# non-noisy pulse, the noisy pulse, and the pulse de-noised with the autoencoder to show the value
+# of the technique.
+# NOTE: The autoencoder .linn file was created using the 
```

Comment:
Using the what? Using "this"?

Reply:
Good catch.  I separated the matched filter code out from the API code, so this NOTE was no longer applicable.  Deleted.